scripts/aws-delete-lamba-layer-versions.sh

```diff
@@ -8,7 +8,7 @@ set -euo pipefail
 # override default AWS region
 export AWS_REGION=eu-central-1
 
-LAYER_NAME=SentryPythonServerlessSDKLocalDev
+LAYER_NAME=SentryPythonServerlessSDK-local-dev
```

Comment:
Please check if the layer name is used also in the Sentry.io backend for setting up Lambda Functions integrations.

Reply:
the layer name is in craft
https://github.com/getsentry/sentry-python/blob/2bb95b3eaa9002f1367cd2600c33350b97aa8f8c/.craft.yml#L12